packages/lesswrong/components/community/modules/ToggleButtonGroup.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { createStyles, withStyles } from '@material-ui/core/styles';
+
+/**
+ * reimplementation of the ToggleButtonGroup from material-ui
+ * with the styles of the DistanceUnitToggle.
+ *
+ * should be replaced with the official material-ui component
+ * after update to material-ui v5.
+ *
+ *
+ */
+
+const styles = createStyles((theme: ThemeType): JssStyles => ({
+  root: {
+    display: 'inline-block',
+    ...theme.typography.commentStyle,
+  },
+}))
+
+const ToggleButtonGroup = (
+  {classes, className = '', children, value, onChange, name}:{classes: ClassesType, className?: string, children: any, value: string, onChange: Function, name?: string}
```

Comment:
I think `value` should be optional, since there might not be a default button selected

Reply:
Good point, but now I'm wondering, should it really be optional, or should it rather be required but allow `null` as value for explicitly signifying the "no value selected" case?